buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -358,10 +360,10 @@ class BeamModulePlugin implements Plugin<Project> {
         google_api_services_storage                 : "com.google.apis:google-api-services-storage:v1-rev124-$google_clients_version",
         google_auth_library_credentials             : "com.google.auth:google-auth-library-credentials:$google_auth_version",
         google_auth_library_oauth2_http             : "com.google.auth:google-auth-library-oauth2-http:$google_auth_version",
-        google_cloud_core                           : "com.google.cloud:google-cloud-core:1.0.2",
-        google_cloud_core_grpc                      : "com.google.cloud:google-cloud-core-grpc:$grpc_version",
+        google_cloud_core                           : "com.google.cloud:google-cloud-core:$google_cloud_core_version",
```

Comment:
google-cloud-core depended on joda-time:2.9.2 at version 1.0.2 also, so I don't think it will be a problem. It turns out that neither google-cloud-core nor google-cloud-spanner actually reference joda-time in code anywhere even though they declare a dependency on it; the actual references are in other clients that Beam doesn't use (google-cloud-bigquery, google-cloud-resourcemanager, google-cloud-dns, google-cloud-compute)